pa_config/performance-analyzer.properties

```diff
@@ -25,7 +25,7 @@ https-enabled = false
 # Setup the correct path for server certificates
 certificate-file-path = specify_path
 private-key-file-path = specify_path
-trusted-cas-file-path = specify_path
+#trusted-cas-file-path = specify_path
```

Comment:
While debugging we had to change the config file at 2 places. While it might be possible that same config file is kept at 2 places, but something we can check before merging.

Reply:
Right, so we copy this file to the 2 locations.

Reply:
Right, so the same file gets copied at the 2 locations.